hat/assets/js/cypress/integration/13 - workflows/list.spec.js

```diff
@@ -0,0 +1,376 @@
+
+            describe('Buttons', () => {
+                it('Delete should only be present on DRAFT version', () => {
+                    cy.get('table tbody tr')
+                        .eq(0)
+                        .find('td')
+                        .eq(2)
+                        .should('contain', 'Draft');
+
+                    cy.get(
+                        '[data-test="delete-dialog-button-workflow-version-12"]',
+                    ).should('be.visible');
+
+                    cy.get('table tbody tr')
+                        .eq(1)
+                        .find('td')
+                        .eq(2)
+                        .should('contain', 'Published');
+
+                    cy.get(
+                        '[data-test="delete-dialog-button-workflow-version-11"]',
+                    ).should('not.exist');
+                });
+                it('Publish should only be present on DRAFT or UNPUBLISHED version', () => {
+                    cy.get('table tbody tr')
+                        .eq(0)
+                        .find('td')
+                        .eq(2)
+                        .should('contain', 'Draft');
+                    cy.get('[data-test="publish-button-12"]').should(
+                        'be.visible',
+                    );
+
+                    cy.get('table tbody tr')
+                        .eq(1)
+                        .find('td')
+                        .eq(2)
+                        .should('contain', 'Published');
+
+                    cy.get('[data-test="publish-button-11"]').should(
+                        'not.exist',
+                    );
+
+                    cy.get('table tbody tr')
+                        .eq(2)
+                        .find('td')
+                        .eq(2)
+                        .should('contain', 'Unpublished');
+
+                    cy.get('[data-test="publish-button-10"]').should(
+                        'be.visible',
+                    );
+                });
+                it('Copy should only be present on DRAFT or UNPUBLISHED version', () => {
+                    cy.get('table tbody tr')
+                        .eq(0)
+                        .find('td')
+                        .eq(2)
+                        .should('contain', 'Draft');
+
+                    cy.get('[data-test="copy-button-12"]').should('not.exist');
+
+                    cy.get('table tbody tr')
+                        .eq(2)
+                        .find('td')
+                        .eq(2)
+                        .should('contain', 'Unpublished');
+
+                    cy.get('[data-test="copy-button-10"]').should('be.visible');
+
+                    cy.get('table tbody tr')
+                        .eq(1)
+                        .find('td')
+                        .eq(2)
+                        .should('contain', 'Published');
+
+                    cy.get('[data-test="copy-button-11"]').should('be.visible');
+                });
+            });
```

Comment:
I think we could regroup these 2 tests. This would improve perf since it would skip a `cy.visit()`